next/pages/[...slug].tsx

```diff
@@ -191,8 +191,8 @@ export const getStaticProps: GetStaticProps<PageProps> = async ({
     typeof previewData.resourceVersion === "string" &&
     previewData.resourceVersion !== "rel:latest-version"
   ) {
-    const [nodeId, revisionId] = previewData.resourceVersion.split(":");
-    const path = `/node/${nodeId}/revisions/${revisionId}/view`;
+    const [_, revisionId] = previewData.resourceVersion.split(":");
+    const path = `/node/${node.id}/revisions/${revisionId}/view`;
```

Comment:
the resourceVersion property looks like this: 

"id:20" for a revision with id 20. 

So the existing code was just wrong :)